sourced/ml/transformers/basic.py

```diff
@@ -225,13 +237,18 @@ def create_parquet_loader(session_name, repositories,
     return parquet
 
 
-def create_uast_source(args, session_name, select=HeadFiles):
+def create_uast_source(args, session_name, select=HeadFiles, language_selector=None,
+                       extract_uast=True):
```

Comment:
Please add docs. It's not clear why it should return `root` and `start_point`. 
`extract_uast` and other arguments are not used in case of `parquet` - maybe it's better to split this function.
It depends on `add_repo2_args` where `parquet` flag is added - so it should be checked that `args` has `parquet` or it should be mentioned that `add_repo2_args` should be used on a parser.